src/core/EdFi.DataManagementService.Core.External/Backend/UpsertResult.cs

```diff
@@ -38,8 +38,7 @@ public record UpsertFailureIdentityConflict(string ReferencingDocumentInfo) : Up
     /// <summary>
     /// A transient failure due to a transaction write conflict
     /// </summary>
-    /// <param name="FailureMessage">A message providing failure information</param>
-    public record UpsertFailureWriteConflict(string FailureMessage) : UpsertResult();
+    public record UpsertFailureWriteConflict() : UpsertResult();
```

Comment:
Removed message because it should be entirely core-provided on a write conflict.